src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/galleryboxes/SpecialUnusedFilesPage.java

```diff
@@ -0,0 +1,15 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.galleryboxes;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+final public class SpecialUnusedFilesPage extends GalleryBox {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)